src/components/Profile/components/PersonalInfo/styles/PersonalInfo.styles.tsx

```diff
@@ -0,0 +1,124 @@
+import { Button, TextField } from '@mui/material';
+import styled, { keyframes } from 'styled-components';
+
+const shake = keyframes`
+    10%, 90% {
+        transform: translate3d(-1px, 0, 0);
+    }
+
+    20%, 80% {
+        transform: translate3d(2px, 0, 0);
+    }
+
+    30%, 50%, 70% {
+        transform: translate3d(-4px, 0, 0);
+    }
+
+    40%, 60% {
+        transform: translate3d(4px, 0, 0);
+    }
+`;
+
+export const ShakingInput = styled(TextField)`
+    &.shake {
+        animation: ${shake} 0.5s ease-in-out;
+    }
+`;
+
+const CustomButton = styled(Button)`
+    width: 50% !important;
+    height: 40px !important;
+    color: white !important;
+    margin-right: auto !important;
+    margin-left: auto !important;
+    letter-spacing: 3px !important;
```

Comment:
Try to avoid `!important` it is considered a bad practice

Reply:
It is necessary to do !important to overwrite material UI components' styles.

Reply:
Fixed this issue.